auth_matrix/templates/admin/auth/user/change_list.html

```diff
@@ -0,0 +1,27 @@
+{% extends "admin/change_list.html" %}
+
+{% block extrastyle %}
+{{ block.super }}
+<style>
+    .extra {
+        padding: 1em;
+    }
+
+    .is-flex {
+        display: flex;
+    }
+</style>
+{% endblock extrastyle %}
+
+
+{% block pagination %}
+    {{ block.super }}
+    {% if request.user.is_superuser %}
```

Comment:
this does not match the permissions defined on the view itself, which currently only requires staff user permissions

Instead, this should use the proper admin view permission check, see https://docs.djangoproject.com/en/4.2/topics/auth/default/#permissions